sdk/tests/deqp/functional/gles3/es3fPixelBufferObjectTest.js

```diff
@@ -214,8 +214,15 @@ var tcuImageCompare = framework.common.tcuImageCompare;
         gl.enableVertexAttribArray(coordLoc);
         assertMsgOptions(gl.getError() === gl.NO_ERROR, 'enableVertexAttribArray failed ', false, true);
 
-        gl.vertexAttribPointer(coordLoc, 3, gl.FLOAT, false, 0, positions[0]);
-        gl.vertexAttribPointer(colorLoc, 4, gl.FLOAT, false, 0, colors[0]);
```

Comment:
It is not correct to feed data to vertex attribs from client memory in WebGL, we should use ARRAY_BUFFER. In native GLES, it is correct in this way.